docs/README.md

```diff
@@ -1,73 +1,123 @@
+    "product-installments"
+  ]
+},
+```
 
-If you feel compelled to give further details about the block, such as it's **relationship with the VTEX admin**, don't hesitate to use this section.
+Every block in this app has some props in common:
 
-## Customization
+| Block name          | Type      |  Description | Default value |
+| --------------------| ----------|--------------|---------------|
+| `markers`           |`string[]` |Custom markers used to customize parts of the message on siteeditor|`[]`|
+|  `blockClass`  |  `String`  |  Unique  block  ID  to  be  used  in [CSS  customization](https://vtex.io/docs/recipes/style/using-css-handles-for-store-customization#using-the-blockclass-property)  |  `undefined`  |
 
-The first thing that should be present in this section is the sentence below, showing users the recipe pertaining to CSS customization in blocks:
+Example  of usage:
 
-`In order to apply CSS customizations in this and other blocks, follow the instructions given in the recipe on [Using CSS Handles for store customization](https://vtex.io/docs/recipes/style/using-css-handles-for-store-customization).`
+```json
+"product-price-savings#summary": {
+  "props": {
+    "markers": [
+      "discount"
+    ],
+    "blockClass": "summary"
+  }
+},
+```
 
-Thereafter, you should add a single column table with the available CSS handles for that block:
+### Step 3 - Editing the values on Site Editor
 
-| CSS Handles |
-| ----------- |
-| `XXXXX`     |
-| `XXXXX`     |
-| `XXXXX`     |
-| `XXXXX`     |
-| `XXXXX`     |
+Every Product Price message uses the [ICU Message Format](https://format-message.github.io/icu-message-format-for-translators/), which means you can interpolate values and apply some conditions to the message. This makes it possible to fully edit the prices' messages. Every block has its values that can be interpolated in the message and you can check them out in the description of the field on site-editor:
 
-If there are none, add the following sentence instead:
+![image](https://user-images.githubusercontent.com/8443580/77782384-f6896b00-7035-11ea-8808-fb2a5533d1a6.png)
 
-`The component still doesn't have CSS Handles for its specific customization.`
+Another special feature these messages have are `markers`, which gives the possibility to wrap a part of the message in a `span` with its own CSS handle. To use `markers` in your message, do the following:
 
----
+1 - Add to the prop `markers` a marker name, for example: `"markers": ["highlight"]`
+2 - Open the Site Editor and click to edit the block
+3 - In the edition form of the block, fill the field with a text and wrap the piece of a text using the marker as if it were an element. Example: `<highlight>-{savingsPercentage}</highlight>`
+4 - Save the changes
+5 - If you inspect the HTML of the element you will notice that this part of the text is wrapped in a new `span` with its own unique selector: `<span class="vtex-product-price-1-x-savings-highlight">`
```

Comment:
@iaronaraujo, I could not review this part because I first need to understand those Messages thing. I'll schedule a time for us to clarify this.